pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -126,9 +126,6 @@ self: super: builtins.intersectAttrs super {
   # fix errors caused by hardening flags
   epanet-haskell = disableHardening ["format"] super.epanet-haskell;
 
-  # Link the proper version.
-  zeromq4-haskell = super.zeromq4-haskell.override { zeromq = pkgs.zeromq4; };
-
```

Comment:
I believe this entry is not needed any more, but I don't understand the haskell scaffolding well.

Reply:
Yeah, that's right. You can drop it.